include/ebpf_structs.h

```diff
@@ -17,25 +17,22 @@ typedef enum bpf_map_type
     BPF_MAP_TYPE_ARRAY = 2,  ///< Array, where the map key is the array index.
     BPF_MAP_TYPE_PROG_ARRAY =
         3, ///< Array of program fds usable with bpf_tail_call, where the map key is the array index.
-    BPF_MAP_TYPE_PERCPU_HASH = 4,
-    BPF_MAP_TYPE_PERCPU_ARRAY = 5,
-    BPF_MAP_TYPE_HASH_OF_MAPS = 6,
-    BPF_MAP_TYPE_ARRAY_OF_MAPS = 7,
-    BPF_MAP_TYPE_LRU_HASH = 8,
-    BPF_MAP_TYPE_LPM_TRIE = 9,
-    BPF_MAP_TYPE_QUEUE = 10,
-    BPF_MAP_TYPE_LRU_PERCPU_HASH = 11,
-    BPF_MAP_TYPE_STACK = 12,
+    BPF_MAP_TYPE_PERCPU_HASH = 4,      ///< Per-CPU hash table.
+    BPF_MAP_TYPE_PERCPU_ARRAY = 5,     ///< Per-CPU array.
+    BPF_MAP_TYPE_HASH_OF_MAPS = 6,     ///< Hash table, where the map value is another map.
+    BPF_MAP_TYPE_ARRAY_OF_MAPS = 7,    ///< Array, where the map value is another map.
+    BPF_MAP_TYPE_LRU_HASH = 8,         ///< Least-recently-used hash table.
+    BPF_MAP_TYPE_LPM_TRIE = 9,         ///< Longest prefix match table.
```

Comment:
trie instead of table? I know our implementation is hashtable based, but the name of the map has "trie" in it.

Reply:
Done